lib/generator.rb

```diff
@@ -36,12 +37,24 @@ def test_cases
   end
 
   def generate
+    generate_test_file
+    update_version
+  end
+
+  private
+
+  def generate_test_file
     File.open(path_to("#{name.gsub(/[ -]/, '_')}_test.rb"), 'w') do |f|
       f.write ERB.new(File.read(path_to('example.tt'))).result binding
+      FileUtils.chmod('+x', f)
```

Comment:
I gather that this will either be a no-op or will work on Windows, depending on who you ask. The docs don't really say either way. But it definitely shouldn't blow up.

@kotp I'm curious as to why you think we shouldn't change the file mode here?

My argument for changing it here is that it will relieve contributors from the burden of having to remember to run another script or `chmod` it themselves. People will predictably forget to chmod the file and we will have to tell them to update it. Or, people might manually change the file mode in weird ways. This standardizes the file mode automatically.